api/app/controllers/review.controller.ts

```diff
@@ -441,3 +443,37 @@ export const getAllReviewsForAdmin = async (req: Request, res: Response) => {
     }
   }
 };
+
+export const resetReviews = async (req: Request, res: Response) => {
+  const { id } = req.params; // Get the review ID from the request parameters
+
+  try {
+    // Find the review
+    const review = await Review.findOne({ where: { id } });
+
+    if (!review) {
+      return res.status(404).send("Review not found");
+    }
+
+    // Delete the review
+    await Review.destroy({ where: { id } });
+
+    // Reset the transcript
+    await Transcript.update(
```

Comment:
I think we should reset the transcript before deleting the review to avoid race conditions. Also, the transcript operation is dependent on the review info so lets move it up the call stack

Reply:
either ways there is no race condition here